src/components/InformationsBar/CloroplethLegend/CloroplethLegend.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import 'jest-styled-components';
+import { render, screen } from '@testing-library/react';
+import { CloroplethLegend } from '.';
+import { Container, Title, Legend, GradientBar } from './styles';
+
+describe('CloroplethLegend', () => {
+
+    test('Test the Container component style in Choropleth Legend', () => {
+        const { container: container } = render(<Container />);
+        expect(container.firstChild).toHaveStyleRule('bottom', '100px');
+        expect(container.firstChild).toHaveStyleRule('position', 'fixed');
+        expect(container.firstChild).toHaveStyleRule('font-size', '14px');
+        expect(container.firstChild).toHaveStyleRule('width', '305px');
+        expect(container.firstChild).toHaveStyleRule('display', 'flex');
+        expect(container.firstChild).toHaveStyleRule('flex-direction', 'column');
+        expect(container.firstChild).toHaveStyleRule('border-radius', '5px !important');
```

Comment:
I really don't think it is necessary to test the style of the component! This is a over verification to the component  visualization. In this cases we should test just the logic and behaviour. :D

Please, feel free to disagree with me, add your opinion about it here